views/account/profile/index.ejs

```diff
@@ -14,33 +14,28 @@
     >
       <div class="row col-12">
         <main class="px-3">
-          <h1 class="display-4">Goon up Brah.</h1>
-          <h1>Hello World!</h1>
-          <p>This file was served with Express.</p>
-          <p>Here is your information:</p>
-          <ul>
-            <li>First Name: <%= user.first_name %></li>
-            <li>Last Name: <%= user.last_name %></li>
-            <li>Email: <%= user.email %></li>
-            <li>Password: <%= user.password %></li>
-            <li>
-              Settings:
-              <ul>
-                <% var settings = JSON.parse(user.settings);
-                Object.keys(settings).forEach( (key) => { %>
-                <li><%= key %>: <%= settings[key] %></li>
-                <% }); %>
-              </ul>
-            </li>
-          </ul>
-
-          <a
-            href="#"
-            class="btn btn-warning btn-lg"
-            role="button"
-            aria-disabled="true"
-            >Generate</a
-          >
+          <h1 class="display-4">Social Media Accounts</h1>
```

Comment:
I know you probably had this because of Payton's sketch, but I think we can do without the header

Reply:
i also thought this but just now realized the header might also just be a good indicator to the user on which page they're on. so im kind of in between on the idea. aesthetic purposes, it would look great without but with it, it might help the reader be more aware of where they are on the app? what do you guys think @lelekaz @paytonshaltis @sderacy @matyis1

Reply:
That's a fair point but I think we can allow the user to know which page they're on in other ways such as having a (current) text next to the hyperlink or something similar.

If we decide to keep the header then we should place that on each page to keep it consistent.